# Don't modify the she-bang line for a cross-build.
# Otherwise it points to our hostpython (which we do not want)
#
# Signed-Off: Michael 'Mickey' Lauer <mickey@vanille-media.de>

Index: python-2.7.2/src/Lib/distutils/command/build_scripts.py
===================================================================
--- python-2.7.2.orig/src/Lib/distutils/command/build_scripts.py	2010-01-23 04:23:15.000000000 -0500
+++ python-2.7.2/src/Lib/distutils/command/build_scripts.py		2010-12-26 10:07:35.762597668 -0500
@@ -85,7 +85,7 @@
                     continue
 
                 match = first_line_re.match(first_line)
-                if match:
+                if False: #match:
                     adjust = 1
                     post_interp = match.group(1) or ''
 
